internal/output/html.go

```diff
@@ -335,11 +381,89 @@ func buildHTMLResult(ecosystemMap map[string][]HTMLSourceResult, resultCount HTM
 	})
 
 	ecosystemResults = append(ecosystemResults, osResults...)
+	var layers []LayerInfo
+	if isContainerScanning {
+		layers = getAllLayers(ecosystemResults)
+	}
+	vulnTypeCount := getVulnTypeCount(ecosystemResults)
 
 	return HTMLResult{
-		EcosystemResults: ecosystemResults,
-		HTMLVulnCount:    resultCount,
+		EcosystemResults:    ecosystemResults,
+		HTMLVulnCount:       resultCount,
+		IsContainerScanning: isContainerScanning,
+		AllLayers:           layers,
+		VulnTypeCount:       vulnTypeCount,
+	}
+}
+
+func getVulnTypeCount(result []HTMLEcosystemResult) VulnTypeCount {
+	osType := 0
+	projectType := 0
+	uncalled := 0
+	all := 0
+
+	for _, ecosystem := range result {
+		for _, source := range ecosystem.Sources {
+			if ecosystem.IsOS {
+				osType += source.HTMLVulnCount.Called
+			} else {
+				projectType += source.HTMLVulnCount.Called
+			}
+			all += source.HTMLVulnCount.Called
```

Comment:
question: is this actually "all called"?

suggestion: you could just calculate the value when returning the struct by doing `osType + projectType`